### PR TITLE
Update calculation of workspace files

### PR DESCRIPTION
**Pull request by dblandin**
Extract file workspace calculation and use `os.walk`.

### CODE REVIEW

File: workspace.py [hunk 1]
```diff
@@ -0,0 +1,40 @@
+import glob
```
Comment (antiagainst) [inline workspace.py]: `glob` and `sys` are not used in this source file. We can also delete them from the main script file if not used.

**Review by antiagainst — CHANGES_REQUESTED** [review]

File: workspace.py (hunk 1 above)
Comment (dblandin) [reply workspace.py]: Good catch! Updated :+1:

**Comment by dblandin** [comment]
I believe this addresses https://github.com/antiagainst/codeclimate-cppcheck/issues/2 but it looks like cppcheck will emit issues for header files, even if the header paths are not included in the provided file list.

Trying to figure out a way around this...

**Review by antiagainst — APPROVED** [review]

**Comment by antiagainst** [comment]
The code LGTM now, but I haven't checked it on CodeClimate CLI. Do you want to solve the problem in another PR? I can merge it right now.

**Comment by dblandin** [comment]
@antiagainst Yep, feel free to merge. I'll open up another PR with some updates.

**Pull request merged**